akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -227,9 +227,6 @@ private[remote] class Remoting(_system: ExtendedActorSystem, _provider: RemoteAc
       s"Attempted to quarantine address [$remoteAddress] with uid [$uid] but Remoting is not running", null)
   }
 
-  // Not used anywhere only to keep compatibility with RemoteTransport interface
-  protected def useUntrustedMode: Boolean = provider.remoteSettings.UntrustedMode
-
```

Comment:
what would break? I don't think anyone extended RemoteTransport since 2.4 at least (in 2.2, maybe also 2.3 you could still configure a different Remoting but this is not exposed anymore). If they did, what did they do with it? It is not possible to load it into Akka. I think this is fine to remove.